frontend/src/services/sc-api/sc-api.service.ts

```diff
@@ -63,6 +66,26 @@ class SCApi {
     );
   }
 
+  public async updateInstance(
+    payload: SCInstanceUpdateParamsDto & SCInstanceUpdateRequestDto,
+  ): Promise<SCInstanceUpdateResponseDto> {
```

Comment:
I'd rather split this param into two params:
```suggestion
  public async updateInstance(
    params: SCInstanceUpdateParamsDto,
    payload: SCInstanceUpdateRequestDto,
  ): Promise<SCInstanceUpdateResponseDto> {
```
It improves code readability imho